Orange/widgets/unsupervised/owcorrespondence.py

```diff
@@ -108,7 +107,7 @@ def set_data(self, data):
         self.data = data
         if data is not None:
             self.varlist[:] = [var for var in data.domain.variables
-                               if var.is_discrete]
+                               if var.is_discrete and len(var.values) > 1]
```

Comment:
This only filters out variables with less than 2 values.
A variable can have 3 values, but after selecting a subset only instances with 1 will remain.
This widget will still crash.